### DISCUSSION TITLE
Antenna Orientation in USRP B210s

### DISCUSSION THREAD

**Discussion by rolandmartinson7890** [srsran/srsRAN_Project#194] [General Help] [answered]
Good day srsRAN users

I would like to ask a question which I think is very important for our community since many people use the B210 for SDR experiments with srsRAN. The question follows up from this discussion which did not conclude the rightful orientation of antennas (graphically): https://github.com/srsran/srsRAN_Project/discussions/102. I would thus like to ask if this is the best orientation (attached below). If not, could someone upload a picture with the right orientation that can be used as reference by people in our community.

Thanks for your help.

![20230831_082203](https://github.com/srsran/srsRAN_Project/assets/142294363/6ab3d50d-7eec-4cb0-84ac-f02c5dc34a20)

**Comment by herlesupreeth** [comment]
I would say the Tx antennas are correctly oriented. For the Rx, the two antennas in the middle, I would keep it facing towards you. So the antenna configuration from left to right will be as follows.

(up (like how it is now), facing towards you (like how it is now), **facing towards you**, right (like how it is now))

So you just need to change one antenna position thats all

**Reply by rolandmartinson7890** [reply]
Thanks for the reply @herlesupreeth. So I am guessing this picture should close the discussion:

![20230831_095331](https://github.com/srsran/srsRAN_Project/assets/142294363/89cfe473-18ec-4e33-aabc-e7a2a6b9e8f1)

**Reply by herlesupreeth** [reply, marked as answer]
Yes, thats right

**Reply by rolandmartinson7890** [reply]
Thanks a lot. I hope this gets to help many people as well.

**Reply by gustavobsch** [reply]
Did you notice any speed gain or signal levels enhancement?

**Reply by rolandmartinson7890** [reply]
Hello @gustavobsch . I wasn't looking for the speed gain but it was something that was just on my mind and I thought would be useful here. I am assuming there would be a slight increase in speed. The bottleneck in my setup is my WIFI so I can not get anything above 30M (so there wasn't a change) since that's the max for my WLAN. Since the picture makes it easier now, someone can add to the discussion by a quick set up.

**Reply by frankist** [reply]
Placing the Tx antennas orthogonal to each other is generally a good idea to leverage spatial diversity and spatial multiplexing. However, in the context of an indoor lab with very short-range communications, there are other phenomena like multipath and near-field effects that also play an important role. It isn't unusual, for me at least, to get higher SNR with antenna configurations different than the supposedly theoretically optimal. So, if all you want is to achieve the maximum rate in your very specific static setup, my general suggestion would be to play around with different configurations and see which one provides you with the best SNR/CQI/RI.

Also, remember that the optimal gNB antenna configuration is strongly dependent on the antenna configuration and position of the UE. If you are holding the COTS UE in your hand, away from the window, the USRP Rx antennas of your last photo will both have their nulls pointing toward you, which may not be ideal for the UL channel. If you are trying to leverage Rx diversity, I would also avoid having the Rx antennas parallel to each other.
